### DISCUSSION TITLE
Configuration reloading - add more reload services, redesign card

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#11226] [Other feature requests]
## The request

Configuration reloading card under Server Control panel is currently missing a bunch of things since a lot more reload services have been added lately. Here are some not currently on the card:

`input_boolean.reload`
`input_datetime.reload`
`input_number.reload`
`input_select.reload`
`input_text.reload`
`frontend.reload_themes`
`person.reload`
`python_script.reload`
`timer.reload`
`zone.reload`

Although it doesn't necessarily require a redesigned card to add these, the current style of every button on a seperate line would result in a very tall height (included an image at the bottom). So here is an idea I had to accommodate these:

![reload-config](https://i.imgur.com/5Ekd6jg.jpg)

Chips with multi-select would allow adding all of the various services while still keeping the card a reasonable size. Even with everything added, it would be shorter than it currently is. An added benefit is that you can reload multiple things at once. So if you make a brand new automation that has an input boolean you could do them both at the same time.

## The alternatives

Alternative would be to keep the existing style but just add everything to it, which would result in a super tall card. Below is a mockup of that, and it's missing stuff like Reload Zones which was just added recently (so it'd become even taller over time). 

![tall](https://i.imgur.com/yCLJSEb.png)

**Comment by Misiu** [comment]
@SeanPM5 another option would be to group all inputs into one group and show one button `RELOAD INPUTS` instead of five, but I think the first idea (multi-select chips) is much better, as it will look better when new reload services will be added.

What do you think about adding a second button `RELOAD ALL`? In both cases clicking on button/buttons should display a confirmation dialog (can't check right now if this already happens)

**Comment by frenck** [comment]
From a user perspective, why not just a single "Reload" button?

**Comment by SeanPM5** [comment]
I don't think it should be a single button because of lower powered hardware. 

With my RPi3 (and 1200 entities) it takes like 7-8 minutes to restart Home Assistant. I would imagine that reloading everything would probably take a similar amount of time -- or at least be much longer than doing only the portions you need. That's too lengthy of a wait imo if you're doing simple tasks like just wanting to see an updated icon/group/scene. 

But I do like the idea and think it should be added, just not forced. I think a "Reload Selected" and "Reload All" button would work best.

**Comment by iantrich** [comment]
FYI It will likely be "RELOAD INPUTS" and not all of them expanded

**Comment by iantrich** [comment]
@frenck, @balloob did note that he isn't sure we'd handle that well as it currently stands FWIW
https://discordapp.com/channels/330944238910963714/351047592588869643/674752665217007646

**Comment by tdejneka** [comment]
The Multi-Chip proposal is both compact and flexible. In the mockup shown above, I suggest the chips be sorted alphabetically. There should also be  means of quickly selecting all chips (perhaps via an "All" chip?)

**Comment by balloob** [comment]
@iantrich we should just test it and see what exceptions show up on backend and frontend and fix those. We should be able to handle a ton of reloads. Note though that reloading input * might trigger automations

**Comment by hmmbob** [comment]
I just came across this issue when I was looking if anyone ever requested the "reload items" to be ordered alphabetically. If possible, it would make absolute sense (to me, that is) to order the buttons / items in alphabetical order if this work is being implemented :)

**Comment by bramkragten** [comment]
Sorting by name is difficult with translations
